plugins/modules/helm.py

```diff
@@ -358,7 +358,7 @@ def fetch_chart_info(module, command, chart_ref):
 
 def deploy(command, release_name, release_values, chart_name, wait,
            wait_timeout, disable_hook, force, values_files, atomic=False,
-           create_namespace=False, replace=False, skip_crds=False):
+           create_namespace=False, replace=False, skip_crds=False, release_namespace=None):
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.

Reply:
@Akasurde changelog added.

Reply:
thanks.